Sources/Turf/Geometries/Polygon.swift

```diff
@@ -297,4 +297,55 @@ extension Polygon {
             ring[2].longitude == ring[0].longitude
         )
     }
-}
+
+
+    /// Calculates the centroid using the mean of all vertices.
+    /// This lessens the effect of small islands and artifacts when calculating the centroid of a set of polygons.
+    public func centroid() -> LocationCoordinate2D? {
```

Comment:
Polygon already has `area` as a computed property rather than a method, so having `centroid` and `centerOfMass` be computed properties too would be more elegant. There’s probably a point at which a parameterless operation is expensive enough that the method syntax is more appropriate, but `centroid` ends up being a single reduction, which is similar to `area`.

Reply:
Makes sense. I've renamed the two for consistency.